main.py

```diff
@@ -1 +1,24 @@
 #Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
This looks great! One thing I think we could refactor is our loop because with our current while loop, It isn't actually looping ten times! For example:

'''python
   for i in range(10)
'''